packages/frontend/src/server/features/data-availability/summary/get-da-summary-entries.ts

```diff
@@ -97,9 +97,28 @@ export async function getDaSummaryEntries() {
     // Sort by total TVS of DA layers
     .sort((a, b) => b.tvs - a.tvs)
 
-  return entries
+  const ethereumEntry = {
+    slug: ethereumDaLayer.display.slug,
+    name: ethereumDaLayer.display.name,
+    kind: ethereumDaLayer.kind,
+    systemCategory: ethereumDaLayer.systemCategory,
+    usedIn: ethereumDaLayer.bridges.flatMap((bridge) => bridge.usedIn),
+    economicSecurity: economicSecurity[ethereumDaLayer.id],
+    bridges: ethereumDaLayer.bridges[0],
+    tvs: getSumFor(
+      ethereumDaLayer.bridges
+        .flatMap((bridge) => bridge.usedIn)
+        .map((usedIn) => usedIn.id),
+    ),
+  }
```

Comment:
let's move this to separate fn

Reply:
https://github.com/l2beat/l2beat/pull/5779/commits/ee665edeedbd2c6a4a1bd4bc78114f2ee0c3c4bc